go/mysql/handshake_test.go

```diff
@@ -116,7 +116,7 @@ func TestSSLConnection(t *testing.T) {
 	}
 	defer os.RemoveAll(root)
 	tlstest.CreateCA(root)
-	tlstest.CreateSignedCert(root, tlstest.CA, "01", "server", "IP:"+host)
+	tlstest.CreateSignedCert(root, tlstest.CA, "01", "server", "server.domain.com")
```

Comment:
How about `server.example.com`?

That's an intentionally reserved domain which can be used in documentations. See www.example.com.

Reply:
Done, for both tests, and rebased / squashed.